templates/openharmony/entry/src/main/ets/cocos/jsb-adapter/sys-ability-polyfill.js

```diff
@@ -25,7 +25,7 @@
 import display from '@ohos.display';
 import i18n from '@ohos.i18n'
 import deviceInfo from '@ohos.deviceInfo'
-import batteryInfo from '@ohos.batteryInfo';
+import batteryInfo from '@ohos.batteryinfo';
```

Comment:
Why change the name?

Reply:
Different versions of the SDK have different names, the version I'm currently using has the name batteryinfo.